### DISCUSSION TITLE
How to comment out as normal SQL?

### DISCUSSION THREAD

**Discussion by duc-ctm-jacobins-digital** [AltimateAI/vscode-dbt-power-user#198] [Q&A] [answered]
Although I want the file to be read as "jinja-sql", I want to comment out as normal SQL `-- one line` instead of jinja `{##}`. Is there any way I can configure this through `.vscode/settings.json`?

**Comment by mdesmet** [comment]
Why would you need any configuration to write standard sql?

**Reply by duc-ctm-jacobins-digital** [reply]
I want to keep some comments after compiling, something like this
![image](https://user-images.githubusercontent.com/70874681/202085709-797c3b97-c6a6-4a00-b200-433f97124331.png)

I just wonder if I can change something in settings.json. I found this thread but it seems I must edit the extension code, which is unstable https://superuser.com/questions/1476090/how-to-customize-comment-character-in-visual-studio-code

**Reply by mdesmet** [reply]
When using command+' for the compile preview, the comments are kept.

<img width="1230" alt="image" src="https://user-images.githubusercontent.com/9265503/202120378-c433cca7-0679-4389-b361-431aa397a61a.png">

Maybe you should go through all the reproduction steps instead of proposing a solution.

**Reply by duc-ctm-jacobins-digital** [reply]
ah sorry I did not explain well. I mean if I use VS Code comment block (Ctrl+/), then it will turn to jinja comment `{# some comment #}`. Later when compiling these comments will be disapeared. If I want to comment as SQL `-- comment`, I must write manually

**Reply by mdesmet** [reply, marked as answer]
I think that feature is provided by the Better Jinja extension. Not sure if it is configurable.

**Reply by rightx2** [reply]
When I tried to uninstall better jinja, I can't because dbt power user depends on it.... I'd like to evade `{# some comment #}` comment style..

**Reply by Gonzalo933** [reply]
Did you figure out if this is possible?

**Comment by Jeroendevr** [comment]
What I think the OP did was:
Uninstalling the Better Jinja extension and setting the file association back to 'SQL' from 'Jinja-sql' makes the comments `--` instead of `{#   #}`. 
This made it work for me.
